services/terrain/doc/install.md

```diff
@@ -74,41 +74,41 @@ modify this file so that it points to the correct hosts.
 
 ## Logging Configuration
 
-The logging settings are stored in `/etc/donkey/log4j.properties`.  The file
+The logging settings are stored in `/etc/terrain/log4j.properties`.  The file
 looks like this by default:
 
 ```properties
 log4j.rootLogger=WARN, A
 
-# Uncomment these lines to enable debugging in Donkey itself.
-# log4j.category.donkey=DEBUG, A
-# log4j.additivity.donkey=false
+# Uncomment these lines to enable debugging in Terrain itself.
+# log4j.category.terrain=DEBUG, A
+# log4j.additivity.terrain=false
 
 # Uncomment these lines to enable debugging in iPlant Clojure Commons.
 # log4j.category.clojure-commons=DEBUG, A
 # log4j.additivity.clojure-commons=false
 
 # Either comment these lines out or change the appender to B when running
-# Donkey in the foreground.
+# Terrain in the foreground.
 log4j.logger.JsonLogger=debug, JSON
 log4j.additivity.JsonLogger=false
 
-# Use this appender for logging JSON when running Donkey in the background.
+# Use this appender for logging JSON when running Terrain in the background.
 log4j.appender.JSON=org.apache.log4j.RollingFileAppender
-log4j.appender.JSON.File=/var/log/donkey/json.log
+log4j.appender.JSON.File=/var/log/terrain/json.log
 log4j.appender.JSON.layout=org.apache.log4j.PatternLayout
 log4j.appender.JSON.layout.ConversionPattern=%d{MM-dd@HH:mm:ss} %-5p (%13F:%L) %3x - %m%n
 log4j.appender.JSON.MaxFileSize=10MB
 log4j.appender.JSON.MaxBackupIndex=1
 
-# Use this appender when running Donkey in the foreground.
+# Use this appender when running Terrain in the foreground.
 log4j.appender.B=org.apache.log4j.ConsoleAppender
 log4j.appender.B.layout=org.apache.log4j.PatternLayout
 log4j.appender.B.layout.ConversionPattern=%d{MM-dd@HH:mm:ss} %-5p (%13F:%L) %3x - %m%n
 
-# Use this appender when running Donkey in the background.
+# Use this appender when running Terrain in the background.
 log4j.appender.A=org.apache.log4j.RollingFileAppender
-log4j.appender.A.File=/var/log/donkey/donkey.log
+log4j.appender.A.File=/var/log/terrain/terrain.log
 log4j.appender.A.layout=org.apache.log4j.PatternLayout
 log4j.appender.A.layout.ConversionPattern=%d{MM-dd@HH:mm:ss} %-5p (%13F:%L) %3x - %m%n
 log4j.appender.A.MaxFileSize=10MB
```

Comment:
@dennisroberts71: May as well get rid of these example configs too when the docs are overhauled.